server/controllers/emptyBuildings/get.js

```diff
@@ -0,0 +1,10 @@
+const { get } = require('../../models/queries/emptyBuildings');
+
+module.exports = async (req, res, next) => {
+  try {
+    const buildings = await get();
+    res.json({ data: buildings, statusCode: 200 });
```

Comment:
```suggestion
    res.json({ statusCode: 200, message: 'Get Empty Buildings', data: buildings });
```

Reply:
i think there is no need for a message in this response